compute/instances.go

```diff
@@ -961,17 +984,14 @@ func (api *_Instance) toNative() (*Instance, error) {
 
 // toTags() injects its state information into a Tags map suitable for use to
 // submit an API call to the vmapi machine endpoint
-func (mcns *InstanceCNS) toTags(m map[string]interface{}) {
-	if mcns.Disable != nil {
-		s := fmt.Sprintf("%t", mcns.Disable)
-		m[CNSTagDisable] = &s
+func (cns *InstanceCNS) toTags(m map[string]interface{}) {
+	if cns.Disable {
+		m[CNSTagDisable] = cns.Disable
```

Comment:
A `bool` `bool` since the JSON encoder will keep it a boolean. Doesn't look like we had any other examples before this.

I'll add the comment here as well.